distribution/alliance/src/main/filtered-resources/etc/org.codice.ddf.catalog.plugin.security.audit.SecurityAuditPlugin.config

```diff
@@ -17,7 +17,5 @@ auditAttributes=[ \
   "ext.resource-classification", \
   "ext.resource-classification-system", \
   "ext.resource-releasability", \
-  "ext.resource-dissemination-controls" \
-]
-service.pid="org.codice.ddf.catalog.plugin.security.audit.SecurityAuditPlugin"
-
+  "ext.resource-dissemination-controls", \
```

Comment:
✏️  Remove trailing `,`.

Reply:
I decided to leave the extra comma and include a note in the Documentation. See https://github.com/codice/ddf/pull/2241/commits/5e3264137917b6ce6ffed300a59c64405933b8e4#r135342949.